### DISCUSSION TITLE
Is there a way to show a module section only on active monitor

### DISCUSSION THREAD

**Discussion by DusanLesan** [Alexays/Waybar#3323] [Q&A]
I would like to mimic my current dwm status bar setup on waybar. 

The left side that describes the state of the current monitor should be distinct to each of the outputs as it is right now.
However, the right side should not be doubled between all of the outputs. I want only active monitor to show the modules in the right section.

How to do that?

**Comment by UtkarshVerma** [comment]
Did you figure out a solution for this? I am also interested in this.

**Reply by DusanLesan** [reply]
No. Because of this, I’ve pretty much ditched all my attempts to use Wayland and returned to my setup with dwm and its own bar using dwmblocks-async
